### PR TITLE
FLICK AA

### PR DESCRIPTION
**Pull request by femboy-boop**
FUCK YOU BAAN

### CODE REVIEW

File: Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp [hunk 1]
```diff
@@ -321,6 +321,70 @@ void CAntiAim::Run(CUserCmd* pCmd, bool* pSendPacket) {
...
+				   //static fake
+				   break;
+				}
+			   else if (pCmd->viewangles.y == 0.f)
+			   {
+				  *pSendPacket == bSendReal == true;
+			   }
+			   else 
+			   {
+				  *pSendPacket == bSendReal == false;
```
Comment (johnathon-walnut) [inline Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp:347]: bruh what does this meannnnnnn what does it do i ddont get it

**Comment by PeterPumpkinEater69real** [comment]
I agree with this pull.

File: Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp (hunk 1 above)
Comment (femboy-boop) [reply Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp:347]: IT SENDS PACKET BACK

**Comment by johnathon-walnut** [comment]
sex

**Comment by johnathon-walnut** [comment]
cba to test

**Comment by lnx00** [comment]
bad code

**Pull request merged**